src/pages/Dashboard.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div>
+    <router-link to="/">Home</router-link>
+    <h1>ダッシュボード</h1>
+    <div v-if="userState.isSignedIn">
+      <p>{{userState.name}}さん、こんにちは！</p>
```

Comment:
マスタッシュ構文では下記のようにスペースを開けるようにしましょう。
他の部分でも同様です。
```suggestion
      <p>{{ userState.name }}さん、こんにちは！</p>
```